website/docs/r/cognito_user_pool.markdown

```diff
@@ -86,10 +86,10 @@ The following arguments are supported:
 
 #### Schema Attributes
 
-  * `attribute_data_type` (Optional) - The attribute data type.
+  * `attribute_data_type` (Required) - The attribute data type. Must be one of `Boolean`, `Number`, `String`, `DateTime`.
   * `developer_only_attribute` (Optional) - Specifies whether the attribute type is developer only.
   * `mutable` (Optional) - Specifies whether the attribute can be changed once it has been created.
-  * `name` (Optional) - The name of the attribute.
+  * `name` (Required) - The name of the attribute.
```

Comment:
Since we put that as Required (which is a good catch by the way), we should also change the related code attribute from Optional to Required

Reply:
Code updated.